app/models/book.py

```diff
@@ -1,11 +1,7 @@
-class Book:
-    def __init__(self, id, title, description):
-        self.id = id
-        self.title = title
-        self.description = description
+from sqlalchemy.orm import Mapped, mapped_column
+from ..db import db
 
-books = [
-    Book(1, "Fictional Book", "A fantasy novel set in an imaginary world."),
-    Book(2, "Wheel of Time", "A fantasy novel set in an imaginary world."),
-    Book(3, "Fictional Book Title", "A fantasy novel set in an imaginary world.")
-]
\ No newline at end of file
+class Book(db.Model):
+    id: Mapped[int] = mapped_column(primary_key=True, autoincrement=True)
+    title: Mapped[str]
+    description: Mapped[str]
```

Comment:
You had mentioned there being implications about `Optional` and nullability on the types, which I hadn't considered here. These will be marked non-nullable, which is probably fine, but we'll need to be sure to mention this in the text since the previous nullability default was effectively to make things nullable. I think this might impact some verbiage in the task list project as well?

Reply:
Exactly, non-nullable is the default with this syntax, so we will need to make sure to update language about nullability in the lesson and task list.